zipkin-storage/cassandra/src/test/java/zipkin2/storage/cassandra/internal/call/ResultSetFutureCallTest.java

```diff
@@ -17,12 +17,11 @@
 import com.datastax.driver.core.exceptions.BusyConnectionException;
 import com.datastax.driver.core.exceptions.BusyPoolException;
 import com.datastax.driver.core.exceptions.QueryConsistencyException;
+import com.google.common.util.concurrent.SettableFuture;
```

Comment:
Intentional to leave this one?

Reply:
yeah this version of cassandra uses guava futures, but that becomes a somewhat mechanical change later.